src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto font', sans-serif;
+}
+
+.heading {
+  text-align: center;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  background-color: #f8f8f8;
+}
+
+.logo {
+  margin: 0 40px;
+  width: 40px;
+  height: 40px;
+  left: 50px;
```

Comment:
Using 'left' in this context might not have the intended effect since '.logo' is not positioned absolutely or relatively. Consider removing it or ensuring the positioning context is set.